changelogs/fragments/546-helm-install-add-support-for-set-options.yaml

```diff
@@ -0,0 +1,4 @@
+---
+minor_changes:
+  - helm - add support for -set-file, -set-json, -set and -set-string options when running helm install (https://github.com/ansible-collections/kubernetes.core/issues/533).
```

Comment:
Is this a leftover?

Reply:
no, this is part of the pull request, along with new module `helm_get` and same feature added for `helm_template`

Reply:
Ok, then I guess PR number should be https://github.com/ansible-collections/kubernetes.core/issues/546 ?

Reply:
no this is the issue #533 which requests it for `helm` module, I add it also for `helm_template`

Reply:
Ah, ok! Thank you.